modules/core/src/main/scala/org/scalasteward/core/buildtool/scalacli/ScalaCliAlg.scala

```diff
@@ -42,8 +63,8 @@ final class ScalaCliAlg[F[_]](implicit
   override def containsBuild(buildRoot: BuildRoot): F[Boolean] = {
     val buildRootPath = buildRoot.relativePath.dropWhile(Set('.', '/'))
     val extensions = Set(".sc", ".scala")
-    gitAlg
-      .findFilesContaining(buildRoot.repo, "//> using lib ")
+    ScalaCliAlg.directives
+      .flatTraverse(gitAlg.findFilesContaining(buildRoot.repo, _))
```

Comment:
I guess there is probably a more efficient way to do this, e.g. using one command, but I didn't spend too long looking into this

Reply:
Yeah, `git grep` (which is used by `findFilesContaining`) can search for multiple patterns in one go by combining them via [`--or`](https://git-scm.com/docs/git-grep#Documentation/git-grep.txt---or).

I just measured the wall clock time of `containsBuild` before and with this change for this repo and it went from 7 to 82 milliseconds. IMO this is acceptable since it is still dwarfed by actually running any build tool to extract dependencies.